cookbook_se/README.md

```diff
@@ -38,6 +38,15 @@ Registration is the second step of the process and it is merely sending those pr
 
 While it's not required to use the git sha for the version, it is highly recommended. If you are registering to your local Docker Desktop Flyte cluster, both the serialization and the registration command have been combined into `make register_sandbox`.
 
+### Fast serialization and registration
+When you make changes that affect code and not dependencies or other container attributes you can use fast registration to avoid having to rebuild your docker container between iterations.
+Like above, the fast iteration process requires serialization and registration using two separate commands.
+
+Serialization is as simple as running `make fast_serialize_sandbox`.
+
+To register and serialize in one go simply run `make fast_register_sandbox`. The output of this command will show entities registered under `--local-source-root` scoped by `--pkgs` with a version of the form `fast...`. Simply execute
```

Comment:
we need to explain how to register with a remote instance

Reply:
you mean using make targets? or with just the command and specifying a different host than localhost?